ledger/api-server-damlonx/reference-v2/BUILD.bazel

```diff
@@ -87,3 +100,28 @@ client_server_test(
         "exclusive",
     ],
 ) if not is_windows else None
+
+client_server_test(
+    name = "conformance-test-h2database",
+    timeout = "long",
+    client = "//ledger/ledger-api-test-tool:ledger-api-test-tool",
+    client_args = [
+        "--all-tests",
+        "--exclude TimeIT",
+    ],
+    data = [
+        "//ledger/test-common:dar-files",
+    ],
+    server = ":plain-reference-server",
+    server_args = [
+        "--jdbc-url jdbc:h2:mem:daml_on_x;db_close_on_exit=false",
+    ],
+    server_files = [
+        "$(rootpaths //ledger/test-common:dar-files)",
+    ],
+    tags = [
+        "dont-run-on-darwin",
+        # NOTE(JM): As this test is somewhat heavy and has timeouts, run it without competition to avoid flakyness.
+        "exclusive",
```

Comment:
We can probably drop this at this point.

Reply:
Ok